pkgs/development/python-modules/flask-security-too/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, flask
+, blinker
+, itsdangerous
+, flask_principal
+, passlib
+, email_validator
+, flask_wtf
+, flask_login
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-Security-Too";
+  version = "4.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16ws5n08vm7wsa2f7lrkxvc7jl3ah1xfylhhyzb4vvqmlk7x9hw8";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    flask_login
+    flask_principal
+    flask_wtf
+    email_validator
+    itsdangerous
+    passlib
+    blinker
+  ];
+
+  doCheck = false;
+
+  checkInputs = [
+    pytest
```

```suggestion
    pytestCheckHook
```
Please use pytestChechkHook everywhere.